.github/workflows/publish.yml

```diff
@@ -1,8 +1,9 @@
 name: publish
 
 on:
-  release:
-    types: [published]
+  push:
+    tags:
+      - 'v*'
```

Comment:
With this change you can locally do

```
$ npm version patch|minor|major
$ git push origin main --follow-tags
```

and GitHub will do the rest 👍🏼

Reply:
@MylesBorins can you update the publish section in the readme with this bit

Reply:
done PTAL